### ISSUE TITLE
Template Engine

### ISSUE THREAD

**Issue by vzuburlis** [GilaCMS/gila#67]
Website users are usually not technical, but they want to make some changes like text in unexpected places in template. An easy, fast and secure way from corrupting the code is needed.

The proposal is make view::render, view::renderFile, view::renderAdmin accept also template files of a template engine. So data is set from the action method with view::set, and then it can be displayed at the file.

An example with mustache php:
```
{{#posts}}
<div>
  <h1>{{title}}</h1>
  <p>{{description}}</p>
  <a href="{{url}}">Read More</a>
</div>
{{/posts}}
```

**Comment by panigrc** [comment]
Well mustache is great but it's logic-less.

I would propose using [twig](https://twig.symfony.com/) which not only offers templating but basic logic, filters and functions and it's also extendable.

**Comment by panigrc** [comment]
Can this commit close? I think you implemented this feature in #68
